staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmss_cache.go

```diff
@@ -35,10 +35,12 @@ var (
 	vmssNameSeparator  = "_"
 	vmssCacheSeparator = "#"
 
+	vmssKey                 = "k8svmssKey"
 	vmssVirtualMachinesKey  = "k8svmssVirtualMachinesKey"
 	availabilitySetNodesKey = "k8sAvailabilitySetNodesKey"
 
 	availabilitySetNodesCacheTTL = 15 * time.Minute
+	vmssTTL                      = 10 * time.Minute
```

Comment:
With the changes that have gone in to [allow unsafe read](https://github.com/kubernetes/kubernetes/pull/83685), the number of ARM calls are reduced. This means we could experiment here by reducing the TTL back to 1 or 2min instead of keeping at 10min. Can we run a test to see what's the impact in reducing this to 1 or 2min instead of keeping it at 10min?

Reply:
> With the changes that have gone in to [allow unsafe read](https://github.com/kubernetes/kubernetes/pull/83685), the number of ARM calls are reduced. This means we could experiment here by reducing the TTL back to 1 or 2min instead of keeping at 10min. Can we run a test to see what's the impact in reducing this to 1 or 2min instead of keeping it at 10min?

@aramase this is a good idea, since you have done such tests before, could you do that on your env again? And we could reduce the TTL back to 1min in master branch first, if necessary, then we could decide whether to cherry-pick.
Current TTL is 10min, as I could find out, it could cause attach disk time cost to 10min in 1.13.12+, 1.14.8+, change back to 1min could mitigate such potential issue. thanks.

Reply:
filed a issue: https://github.com/kubernetes/kubernetes/issues/86125 to track